torch/distributions/bernoulli.py

```diff
@@ -46,6 +46,12 @@ def log_prob(self, value):
         # evaluate using the values
         return log_pmf.gather(-1, value.unsqueeze(-1).long()).squeeze(-1)
 
+    def entropy(self):
+        p = torch.stack([self.probs, 1.0 - self.probs])
+        p_log_p = torch.log(p) * p
+        p_log_p[p == 0] = 0
+        return -p_log_p.sum(0)
```

Comment:
Shouldn't this be summed along the last axis?

Reply:
This sums out over the new dimension created by `torch.stack()` which is dimension 0. The default order for `torch.satck()` and is cheaper than any other dimension since the stacked tensors can be copied contiguously (rather than interleaved).

Reply:
Ahh..missed that. Makes sense.